spec/models/piece_spec.rb

```diff
@@ -5,19 +5,76 @@
   let(:white_player) { FactoryGirl.create(:player, email: 'blah@blah.com', password: 'SPACECAT') }
   let(:black_player) { FactoryGirl.create(:player, email: 'meow@meow.com', password: 'MONORAILCAT') }
   let(:game) { FactoryGirl.create(:game, white_player_id: white_player.id, black_player_id: black_player.id) }
-  let(:piece) { FactoryGirl.create(:piece, x_position: 4, y_position: 4, game_id: game.id, player_id: white_player.id) }
 
-  let(:pawn) { FactoryGirl.create(:pawn, x_position: 2, y_position: 1, game_id: game.id, player_id: black_player.id) }
+  describe "King#valid_move?" do
+
+    let(:king) { FactoryGirl.create(:king, x_position: 4, y_position: 4, game_id: game.id, player_id: black_player.id) }
```

Comment:
Best to just add all these attributes as part of the king factory, allowing them to be overridden if necessary. Otherwise, things start getting nasty when other pieces start coming into play